project.clj

```diff
@@ -18,4 +21,34 @@
               :dependencies [[criterium "0.4.4"]
                              [org.clojure/tools.cli "0.3.3"]
                              [org.clojure/test.check "0.9.0"]
-                             [com.infolace/excel-templates "0.3.3"]]}})
+                             [com.infolace/excel-templates "0.3.3"]]}
+             :dev {:dependencies [[binaryage/devtools "0.8.2"]
+                                  [figwheel-sidecar "0.5.8"]
+                                  [com.cemerick/piggieback "0.2.1"]
+                                  [org.clojure/test.check "0.9.0"]]
+                   :source-paths ["src" "dev"]
+                   ;; need to add dev source path here to get user.clj loaded
+                   ;; TODO check whether this still makes sense,
+                   ;; workaround for cider cljs REPL
+                   #_:figwheel #_{:nrepl-port 7888
```

Comment:
Why is this commented out? I'd love to have everything working in CI, since I'm a cljs novice :)